tests/unit/models/test_output.py

```diff
@@ -37,19 +37,19 @@ class TestComponent:
             ),
             (
                 {"name": "mypkg", "version": "git+https://github.com/myorg/mydep.git"},
-                Component(name="mypkg"),
+                Component(name="mypkg", version="git+https://github.com/myorg/mydep.git"),
             ),
             (
                 {"name": "mypkg", "version": "https://github.com/myorg/mydep.git"},
-                Component(name="mypkg"),
+                Component(name="mypkg", version="https://github.com/myorg/mydep.git"),
             ),
             (
                 {"name": "mypkg", "version": "./a/replacement/module"},
-                Component(name="mypkg"),
+                Component(name="mypkg", version="./a/replacement/module"),
             ),
             (
                 {"name": "mypkg", "version": "file:mypkg"},
-                Component(name="mypkg"),
+                Component(name="mypkg", version="file:mypkg"),
```

Comment:
> Proper validation is handled at package manager level.

:tada: 

IMO we should remove these 4 cases, they were testing the `_valid_semver` validator